test/e2e/tests/nft/view-erc1155-details.spec.js

```diff
@@ -27,41 +36,39 @@ describe('View ERC1155 NFT details', function () {
         ganacheOptions,
         smartContract,
         title: this.test.title,
+        testSpecificMock: mockIPFSRequest,
       },
       async ({ driver }) => {
         await driver.navigate();
         await unlockWallet(driver);
 
         // Click to open the NFT details page and check displayed account
         await driver.clickElement('[data-testid="home__nfts-tab"]');
```

Comment:
I don't see the changes from this PR here https://github.com/MetaMask/metamask-extension/pull/21628

Reply:
those changes were removed on #21639 ([in this line](https://github.com/MetaMask/metamask-extension/pull/21639/files#diff-04273c54389ca3276291f98b1eb04a456c46498dd550f9ee7ba64af9f809fa69L38)) which was also cherry-picked